### PR TITLE
add git hooks

### PR DESCRIPTION
**Pull request by stephenkingsley**
the issue is #22 .

 - [x] add git hooks when commit

I think add some eslint config, and then modify the precommit script.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+dist/
```
Comment (thiamsantos) [inline .gitignore]: For now you have to commit the dist too. Ref #21

Comment (stephenkingsley) [reply .gitignore]: why commit dist? I think push dist to npm is enough.

Comment (thiamsantos) [reply .gitignore]: Yes, I think the same thing. But there is bower, and to publish the dist 
in a automated we need more workaround. [We are solving this](https://github.com/ApoorvSaxena/lozad.js/pull/21#issuecomment-329359790), but  for now just commit the dist folder.

Comment (stephenkingsley) [reply .gitignore]: ok，I reverse this config. and this pr can merge?

Comment (thiamsantos) [reply .gitignore]: Looks good to me. Let's wait for @ApoorvSaxena to approve.

**Comment by ApoorvSaxena** [comment]
👍

**Pull request merged**